README.md

```diff
@@ -1,16 +1,115 @@
 
-Sample app for Tanzu Spring Cloud Gateway tile. 
-Features we demonstrate with this sample app:
+Sample app for VMware's Spring Cloud Gateway commercial products. Features we demonstrate with this sample app:
+
 - Routing traffic to configured internal routes with container-to-container network
 - Gateway routes configured through service bindings
 - Simplified route configuration
 - SSO login and token relay on behalf of the routed services
 - Required scopes on routes (tag: `require-sso-scopes`)
 - Circuit breaker filter
 
-## Deploy to CF
+## Deploy to Kubernetes
+
+The Kubernetes deployment requires you to install [kustomize](https://kustomize.io/). You will also need to install [Spring Cloud Gateway for Kubernetes](https://network.pivotal.io/products/spring-cloud-gateway-for-kubernetes) successfully onto your target Kubernetes cluster.
+
+### Configure Single Sign-On (SSO)
+
+For Animal Rescue sample Single Sign-On (SSO) to work, you will need to create two text files that will be used to create Kubernetes secrets:
+
+* ./frontend/secrets/sso-credentials.txt
+* ./gateway/sso-secret-for-gateway/secrets/test-sso-credentials.txt
+
+Before you start, and for validation, please locate the JWKS endpoint info `jwks_uri` from your SSO identity provider. The endpoint typically exists at:
+
+```
+https://YOUR_DOMAIN/.well-known/openid-configuration
+```
+
+For example, when using Okta the configured Issuer URI and JWKS URI can be retrieved at:
+
+```
+https://<issuer-uri>/.well-known/openid-configuration
+
+$ curl https://dev-1234567.okta.com/oauth2/abcd12345/.well-known/openid-configuration
+
+{
+  "issuer": "https://dev-1234567.okta.com/oauth2/abcd12345",
+...
+  "jwks_uri": "https://dev-1234567.okta.com/oauth2/abcd12345/v1/keys",
+....
+
+# Please note that the format used by Okta is jwks_uri="<issuer-uri>/v1/keys"
+```
+
+The contents of the `./frontend/secrets/sso-credentials.txt` file for example would be the following:
+
+```
+jwks_uri=https://dev-1234567.okta.com/oauth2/abcd12345/v1/keys
+```
+
+The contents of the `./gateway/sso-secret-for-gateway/secrets/test-sso-credentials.txt` file includes the following values from your OpenID Connect (OIDC) compliant SSO identity provider:
+
+```
+scope=openid,profile,email
+client-id={your_client_id}
+client-secret={your_client_secret}
+issuer-uri={your_issuer_uri}
+```
+
+### Deploy with Kustomize
+
+Assuming you are authenticated onto target Kubernetes cluster, you can run the following command from top-level directory in the repository:
+
+```bash
+kustomize build . | kubectl apply -f -
+```
+
+This will create a namespace named `animal-rescue`, create a new gateway instance named `gateway-demo` in that namespace, deploy the frontend and backend Animal Rescue applications and finally apply the application specific API route configurations to `gateway-demo`.
+
+### Deploy with Kubectl
```

Comment:
Is this section incomplete? I don't see any commands here

Reply:
<img width="1156" alt="Screen Shot 2021-03-05 at 9 57 51 AM" src="https://user-images.githubusercontent.com/573621/110154600-48d02180-7d99-11eb-9fff-d013e87c3392.png">

This is all that was in the current deploy_to_k8s.md file.